buku.py

```diff
@@ -4089,14 +4108,16 @@ def main():
 
     search_grp = argparser.add_argument_group(
         title='SEARCH OPTIONS',
-        description='''    -s, --sany           find records with ANY matching keyword
+        description='''    -s, --sany [...]     find records with ANY matching keyword
                          this is the default search option
-    -S, --sall           find records matching ALL the keywords
+    -S, --sall [...]     find records matching ALL the keywords
                          special keywords -
                          "blank": entries with empty title/tag
                          "immutable": entries with locked title
+    -W, --without [...]  combine with keyword search to exclude
```

Comment:
How about using `-x` and `--exclude`?

Reply:
It would eliminate any confusion between small and capital `w`.

Reply:
Good idea.  Changing now.